server/taskflows/hpccloud/taskflow/paraview/visualizer.py

```diff
@@ -182,7 +182,7 @@ def upload_input(task, cluster, job, *args, **kwargs):
     if file_id:
         file_id = file_id[0].value
         task.logger.info('Visualizing file ID: %s' % file_id)
-        job['params']['dataDir'] = '.'
+        job['params']['dataDir'] = '$PWD'
 
```

Comment:
I'm not sure this change is correct. $PWD on the machine running the celery task is likely to be different to $PWD on the cluster that the job is being run on? I think what is why . is being used.